lib/_http_server.js

```diff
@@ -498,6 +498,10 @@ function storeHTTPOptions(options) {
 }
 
 function setupConnectionsTracking(server) {
+  if (!server) {
+    server = this;
+  }
+
```

Comment:
Why this addition?

Reply:
I would prefer to just use `this` without the `server` argument.

Reply:
Yes, it was unnecesssary. @lpinca is right on https://github.com/nodejs/node/pull/48611#discussion_r1248105839 - Removing the argument completely.